documentation/modules/exploit/windows/nimsoft/nimcontroller_bof.md

```diff
@@ -0,0 +1,101 @@
+## Vulnerable Application
+
+All CA Infrastructure Management monitoring agents prior to 9.20 are vulnerable to a buffer overflow vulnerability within the nimcontroller when using the directory_list probe. Since the directory_list probe requires read privileges the target host must also be vulnerable to CVE-2020-8010 to bypass ACL settings. Successful code execution will result in a NT AUTHORITY\SYSTEM shell, even if exploitation fails the remote service will not crash. You should be able to exploit the service an unlimited amount of times.
+
```

Comment:
If the software is available for download somewhere, please add the links here.

Reply:
Unfortunately this software isn't open source or available for free. Would i just add a link to the vendor listing the application?

Reply:
Yeah if the software is not available, then a markdown hyperlink to the vendor website is a good idea.

We'll still need a pcap as part of the verification process. But it is perhaps best to hold off with the pcap until the module is more thoroughly reviewed.